aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -118,6 +122,38 @@ def _is_supported_image(
             log.debug(f"Unhandled exception: {e}")
             return False
 
+    @staticmethod
+    def _guess_ome_dim_order(tiff: TiffFile, ome: OME, scene_index: int) -> List[str]:
+        """
+        Guess the dimension order based on OME metadata and actual TIFF data.
+
+        Parameters
+        -------
+        tiff: TiffFile
+            A constructed TIFF object to retrieve data from.
+        ome: OME
+            A constructed OME object to retrieve data from.
+        scene_index: int
+            The current operating scene index to pull metadata from.
+
+        Returns
+        -------
+        dims: List[str]
+            Educated guess of the dimension order for the file
+        """
+        dims_from_ome = metadata_utils.get_dims_from_ome(ome, scene_index)
+
+        dims_from_tiff_axes = list(tiff.series[scene_index].axes)
```

Comment:
implicit assumption that the dim names coming from tiffffile are compatible with our TCZYXS?

Reply:
Ah yes, I'll comment on this. I'd assumed we were already dependent on that assumption based [on these lines](https://github.com/AllenCellModeling/aicsimageio/blob/main/aicsimageio/readers/ome_tiff_reader.py#L286-L289), but perhaps this comment is just refering to non-TCZYXS dimensions.